### ISSUE TITLE
sklearn-nature-inspired-algorithms is compatible with scikit-learn 0.23.2

### ISSUE THREAD

**Issue by hanzigs** [timzatko/Sklearn-Nature-Inspired-Algorithms#7]
Hi,
Initially I have scikit-learn version
```
Name: scikit-learn
Version: 0.23.2
```
While install "pip install sklearn-nature-inspired-algorithms"
we get scikit-learn version downgraded to 
```
Name: scikit-learn
Version: 0.22.2.post1
```
then i am upgrading scikit-learn back "pip install -U scikit-learn" we get this
```
ERROR: sklearn-nature-inspired-algorithms 0.4.5 has requirement scikit-learn<0.23.0,>=0.22.2, but you'll have scikit-learn 0.23.2 which is incompatible.
```
sklearn-nature-inspired-algorithms works fine with scikit-learn 0.23.2, 
Is it possible for updating the module please.

I am having problem inside docker, with my packages requirements file, scikit-learn got installed first, then sklearn-nature-inspired-algorithms getting installed and this downgrading the scikit-learn which is causing other module issues,

As "sklearn-nature-inspired-algorithms is compatible with scikit-learn 0.23.2" is it possible for updating the module pip please.
Thanks

**Comment by timzatko** [comment]
I have updated all of the dependencies.

https://github.com/timzatko/Sklearn-Nature-Inspired-Algorithms/releases/tag/v0.5.0

**Comment by hanzigs** [comment]
Hi, 
Thanks for the Update, 
this below, numpy version I am using 1.18.5 which is compatible as well and working good, we are getting
```
ERROR: sklearn-nature-inspired-algorithms 0.5.0 has requirement numpy<2.0,>=1.19, but you'll have numpy 1.18.5 which is incompatible.
```
My requirement on tensorflow restricts me with numpy 1.18.5.

This is not mandatory, just for sake of error display while installation.
Thanks

**Comment by timzatko** [comment]
I tried to fix it, please install `v0.5.1`.

**Comment by hanzigs** [comment]
That's Great Thanks
